steps/property-inspection/smoke-alarm.tsx

```diff
@@ -119,7 +119,8 @@ const step = {
               "property"
             >;
           }): boolean {
-            return stepValues["has-smoke-alarm"] === "yes";
+            return stepValues["has-smoke-alarm"] === "yes" ||
+            stepValues["has-smoke-alarm"] === "no"
```

Comment:
You'll need to fix the indentation on this. Run `npm run format` to autofix it, amend the commit, and force push.

Reply:
should be ok now.